src/main/js/dev/rest/localisation/translations_en.json

```diff
@@ -300,28 +301,55 @@
   "registration.times.info": "Registration to the language proficiency test starts at 10.00 and ends at 16.00 o'clock._en",
   "registration.title": "Title of test",
-  "registration.tutorial": "Link to the instruction video",
-  "registration.tutorial.url": "https://www.youtube.com/watch?v=pUVgdF-KBWQ",
   "registration.tutorialVideo": "Instruction video",
   "registry.document.title": "YKI - test organiser register",
+  "registry.examSession.organizationNotFound": "Organiser not found",
+  "registry.examSessions.back": "Back",
   "registry.search.languageSelectPlaceholder": "Select language",
   "registry.search.levelSelectPlaceholder": "Select level",
   "registry.search.noResults": "No organisers found with the submitted search criteria.",
+  "registry.search.orgType01": "Training provider",
+  "registry.search.orgType02": "Academy",
+  "registry.search.orgType05": "Other organisation",
   "registry.search.placeholder": "Search for organiser or location",
+  "registryItem.add.header": "Add new organiser",
+  "registryItem.agreement.addPdf": "Add PDF",
   "registryItem.agreement.addPdfFailed": "The contract could not be added",
   "registryItem.agreement.addPdfSuccess": "The contract was added",
   "registryItem.agreement.loadPdf": "Download the pdf",
-  "registryItem.agreementStart.required": "Start date of agreement period missing."
+  "registryItem.agreementEnd": "End date",
+  "registryItem.agreementEnd.required": "End date is required",
+  "registryItem.agreementExpired": "Agreement is outdated",
+  "registryItem.agreementStart": "Start date",
+  "registryItem.agreementStart.required": "Start date of agreement period missing.",
+  "registryItem.button.add": "Add organiser",
+  "registryItem.button.update": "Save changes",
+  "registryItem.contact": "Contact information",
+  "registryItem.contactEmail": "Email",
+  "registryItem.contactEmail.required": "Contact person email is missing.",
+  "registryItem.contactHeader": "Contact person information",
+  "registryItem.contactName": "Name",
+  "registryItem.contactName.required": "Contact person name is missing.",
+  "registryItem.contactPhone": "Phone number",
+  "registryItem.contactPhone.required": "Contact person phone number is missing.",
+  "registryItem.delete": "Delete organiser",
+  "registryItem.deleteCancel": "Do not delete organiser",
+  "registryItem.deleteConfirm": "Confirm deletion",
+  "registryItem.extra": "Extra information",
+  "registryItem.extra.placeholder": "F.ex. Common mailing list everybody@organiser.fi",
+  "registryItem.merchantId": "Merchant ID",
+  "registryItem.merchantSecret": "Merchant secret",
+  "registryItem.merchantSecret.mandatory": "Mandatory when merchant ID is given",
+  "registryItem.payment": "Payment service information",
+  "registryItem.paymentInfoMissing": "Payment service information missing",
+  "registryItem.search.placeholder": "Search by organisation name",
+  "registryItem.viewAsOrganiser": "Show organiser view"
 }
```

Comment:
The line break after the curly brace could be removed given it doesn't exist in fi, or sv files either. Not that it would really make a difference in comparing their contents / keys against one another.